### PR TITLE
fixing an array/list issue

### PR DESCRIPTION
**Pull request by HenryGeorgist**
found an odd array casting "bug" that is not necessary.

### CODE REVIEW

File: fda-model/paireddata/PairedData.cs [hunk 1]
```diff
@@ -18,7 +18,7 @@ public class PairedData: IPairedData
         /// </summary>
         public double f(double x){
             //binary search.
-            double[] xarr = Xvals.ToArray();
+            double[] xarr = Xvals; //probably not necessary anymore.
```
Comment (Brennan1994) [inline fda-model/paireddata/PairedData.cs:21]: Can we kill this guy all together and just binary search Xvals?

**Review by Brennan1994 — CHANGES_REQUESTED** [review]
love it. Can we just delete that line that makes the new array though?

File: fda-model/paireddata/PairedData.cs (hunk 1 above)
Comment (HenryGeorgist) [reply fda-model/paireddata/PairedData.cs:21]: yep, that would be fine.

**Review by Brennan1994 — APPROVED** [review]
nice

**Pull request merged**